salat-core/src/test/scala/com/novus/salat/test/BinarySpec.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2010 - 2012 Novus Partners, Inc. (http://www.novus.com)
```

Comment:
If it's a brand new file, entirely authored by you - then the copyright belongs to you.

After spinning salat out to its own repo, all new changes are copyright Rose Toomey and individual contributors.

All pre-existing code is copyright Novus Partners.